google-http-client/src/main/java/com/google/api/client/http/HttpHeaders.java

```diff
@@ -158,6 +158,10 @@ public HttpHeaders() {
   @Key("User-Agent")
   private List<String> userAgent;
 
+  /** {@code "Warning"} header. */
+  @Key("Warning")
+  private List<String> warning;
```

Comment:
please rename this variable to `warnings`.

Reply:
Renamed